README.md

```diff
@@ -67,8 +70,44 @@ ros2 launch grid_map_geo load_tif_launch.xml
 ## Running the package
 
 The default launch file can be run as the following command. 
-```
+```bash
 source install/setup.bash
 ros2 launch grid_map_geo load_tif_launch.xml
 ```
 
+To debug the map publisher in GDB:
```

Comment:
Nice!

Reply:
Yea, super helpful for chasing the segfaults in GDAL.